codecov upload failing on windows py37

```
##[section]Starting: Upload coverage to codecov.io
==============================================================================
Task         : Bash
Description  : Run a Bash script on macOS, Linux, or Windows
Version      : 3.151.3
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/bash
==============================================================================
Generating script.
Script contents:
bash <(curl https://codecov.io/bash) -t ***
[command]"C:\Program Files\Git\bin\bash.exe" --noprofile --norc -c pwd
/d/a/_temp
========================== Starting Command Output ===========================
[command]"C:\Program Files\Git\bin\bash.exe" --noprofile --norc /d/a/_temp/069b452e-2aaa-4c0a-8ce4-3748bfd77a29.sh
bash: /dev/fd/63: No such file or directory
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 12 47324   12  5781    0     0  16708      0  0:00:02 --:--:--  0:00:02 16708
curl: (23) Failed writing body (2402 != 4087)
##[error]Bash exited with code '127'.
##[section]Finishing: Upload coverage to codecov.io
```

--- a/ci/azure/unit-tests.yml
+++ b/ci/azure/unit-tests.yml
@@ -19,7 +19,8 @@ steps:
   displayName: Run tests
 
 - bash: |
-    bash <(curl https://codecov.io/bash) -t 688f4d53-31bb-49b5-8370-4ce6f792cf3d
+    curl https://codecov.io/bash > codecov.sh
+    bash codecov.sh -t 688f4d53-31bb-49b5-8370-4ce6f792cf3d
   displayName: Upload coverage to codecov.io
 
 # TODO: publish coverage results to Azure, once we can merge them across

